src/torchcodec/decoders/_video_decoder.py

```diff
@@ -103,6 +103,7 @@ def __init__(
         dimension_order: Literal["NCHW", "NHWC"] = "NCHW",
         num_ffmpeg_threads: int = 1,
         device: Optional[Union[str, torch_device]] = "cpu",
+        transforms: List[Any] = [],  # TRANSFORMS TODO: what is the user-facing type?
```

Comment:
Note: it can be dangerous to use *mutable* values as parameter defaults, because this `[]` object now becomes a global value that can be changed across invocations. If anywhere within `VideoDecoder` we (as developers) were to write something that mutates `tranforms` inplace, we'd be changing the default value:

```py
In [1]: def f(transforms=[]): print(transforms); transforms.append("hello")

In [2]: f()
[]

In [3]: f()
['hello']

In [4]: f()
['hello', 'hello']
```

Best to use `None` or a non-mutable like `tuple()` (we'll probably want to accept sequences, not just lists)